dashboard/test/integration/registration/email_test.rb

```diff
@@ -53,8 +53,109 @@ class EmailTest < ActionDispatch::IntegrationTest
       created_user&.destroy!
     end
 
-    private def finish_email_sign_up(user_type, email)
-      post '/users', params: finish_sign_up_params(user_type: user_type, email: email)
+    test "teacher sign-up" do
```

Comment:
There was previously only a "student sign-up" test, so I added a teacher test for the old flow as well as tests for the new flow.